pkg/apb/client.go

```diff
@@ -158,14 +158,9 @@ func (c *Client) RunImage(
 	c.log.Notice("Login token successfully refreshed.")
 
 	c.log.Debug("Running OC run...")
-	c.log.Debug("clusterConfig:")
-	c.log.Debug("target: [ %s ]", clusterConfig.Target)
-	c.log.Debug("user: [ %s ]", clusterConfig.User)
-	c.log.Debug("password:[ %s ]", clusterConfig.Password)
 	c.log.Debug("name:[ %s ]", spec.Name)
 	c.log.Debug("image:[ %s ]", spec.Image)
 	c.log.Debug("action:[ %s ]", action)
-	c.log.Debug("params:[ %s ]", string(params))
```

Comment:
Params can contain the passwords for an app.  Maybe we can output ****** instead.